src/main/groovy/com/pszymczyk/consul/ConsulProcess.groovy

```diff
@@ -23,6 +23,8 @@ class ConsulProcess implements AutoCloseable {
         this.address = address
         this.process = process
         this.simpleConsulClient = new SimpleConsulClient(address, httpPort)
+
+        addShutdownHook { this.process.destroyForcibly()}
```

Comment:
why do we need this change? `AutoCloseable` does not do the job?

Reply:
Look at my PR: https://github.com/pszymczyk/embedded-consul/pull/77

We can discuss is it covering all cases or maybe shutdown hook is better option.